[2024-09-23 23:20:48,813] INFO _log() L96   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:20:48,817] INFO _log() L96   [33mPress CTRL+C to quit[0m - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:20:48,835] INFO _log() L96    * Restarting with watchdog (windowsapi) - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:20:51,103] WARNING _log() L96    * Debugger is active! - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:20:51,106] INFO _log() L96    * Debugger PIN: 132-689-196 - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:20:51,132] INFO _log() L96   127.0.0.1 - - [23/Sep/2024 23:20:51] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:20:51,265] INFO _log() L96   127.0.0.1 - - [23/Sep/2024 23:20:51] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:20:51,312] INFO _log() L96   127.0.0.1 - - [23/Sep/2024 23:20:51] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:20:51,465] INFO _log() L96   127.0.0.1 - - [23/Sep/2024 23:20:51] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:20:51,585] INFO _log() L96   127.0.0.1 - - [23/Sep/2024 23:20:51] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:20:51,673] INFO _log() L96   127.0.0.1 - - [23/Sep/2024 23:20:51] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:21:03,604] INFO _log() L96    * Detected change in 'C:\\Users\\juan_\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\sklearn\\base.py', reloading - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:21:03,641] INFO _log() L96   127.0.0.1 - - [23/Sep/2024 23:21:03] "POST /paciente HTTP/1.1" 200 - - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:21:04,496] INFO _log() L96    * Restarting with watchdog (windowsapi) - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:21:07,209] WARNING _log() L96    * Debugger is active! - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:21:07,211] INFO _log() L96    * Debugger PIN: 132-689-196 - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:21:12,322] INFO _log() L96   127.0.0.1 - - [23/Sep/2024 23:21:12] "GET /pacientes HTTP/1.1" 200 - - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:21:29,908] INFO _log() L96   127.0.0.1 - - [23/Sep/2024 23:21:29] "[31m[1mPOST /paciente HTTP/1.1[0m" 422 - - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:23:32,983] INFO _log() L96    * Detected change in 'C:\\Users\\juan_\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\streamlit\\__init__.py', reloading - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:23:33,961] INFO _log() L96    * Restarting with watchdog (windowsapi) - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:23:37,317] WARNING _log() L96    * Debugger is active! - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:23:37,320] INFO _log() L96    * Debugger PIN: 132-689-196 - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:24:04,969] INFO _log() L96    * Detected change in 'C:\\Users\\juan_\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\junitxml.py', reloading - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:24:05,723] INFO _log() L96    * Restarting with watchdog (windowsapi) - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:24:08,004] WARNING _log() L96    * Debugger is active! - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:24:08,007] INFO _log() L96    * Debugger PIN: 132-689-196 - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:26:01,557] WARNING add_paciente() L111  Erro ao adicionar paciente 'João': Paciente já existente na base :/ - call_trace=C:\Users\juan_\OneDrive\Área de Trabalho\MVP\app_classifica_problema_cardiaco_api\app.py L111 
[2024-09-23 23:26:01,559] INFO _log() L96   127.0.0.1 - - [23/Sep/2024 23:26:01] "[31m[1mPOST /paciente HTTP/1.1[0m" 409 - - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:29:27,133] INFO _log() L96   127.0.0.1 - - [23/Sep/2024 23:29:27] "POST /paciente HTTP/1.1" 200 - - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:29:37,283] INFO _log() L96   127.0.0.1 - - [23/Sep/2024 23:29:37] "GET /pacientes HTTP/1.1" 200 - - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:30:44,471] INFO _log() L96   127.0.0.1 - - [23/Sep/2024 23:30:44] "POST /paciente HTTP/1.1" 200 - - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:30:48,242] INFO _log() L96   127.0.0.1 - - [23/Sep/2024 23:30:48] "GET /pacientes HTTP/1.1" 200 - - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:32:30,210] INFO _log() L96   127.0.0.1 - - [23/Sep/2024 23:32:30] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:32:30,320] INFO _log() L96   127.0.0.1 - - [23/Sep/2024 23:32:30] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:32:30,322] INFO _log() L96   127.0.0.1 - - [23/Sep/2024 23:32:30] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:32:30,324] INFO _log() L96   127.0.0.1 - - [23/Sep/2024 23:32:30] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:32:30,525] INFO _log() L96   127.0.0.1 - - [23/Sep/2024 23:32:30] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:32:30,617] INFO _log() L96   127.0.0.1 - - [23/Sep/2024 23:32:30] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:32:42,794] WARNING delete_paciente() L139  Erro ao deletar paciente 'ida': Paciente não encontrado na base :/ - call_trace=C:\Users\juan_\OneDrive\Área de Trabalho\MVP\app_classifica_problema_cardiaco_api\app.py L139 
[2024-09-23 23:32:42,795] INFO _log() L96   127.0.0.1 - - [23/Sep/2024 23:32:42] "[33mDELETE /paciente?name=ida HTTP/1.1[0m" 404 - - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:32:53,701] INFO _log() L96   127.0.0.1 - - [23/Sep/2024 23:32:53] "DELETE /paciente?name=Ida HTTP/1.1" 200 - - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-09-23 23:33:11,330] INFO _log() L96    * Detected change in 'C:\\Users\\juan_\\OneDrive\\Área de Trabalho\\MVP\\app_classifica_problema_cardiaco_api\\app.py', reloading - call_trace=C:\Users\juan_\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
